css-brochure-form/styles.css

```diff
@@ -12,7 +12,7 @@ h2 {
   color: #e4561e;
 }
 
-body {
+.body {
```

Comment:
This totally works, but you _really_ don't want a class that is the same name as a tag. That is a one-way ticket to confusion and bugs. Change this name to something else that is still relevant, and you should be good 👍